src/app/__tests__/UpdateServerManager.test.js

```diff
@@ -208,11 +208,12 @@ describe('UpdateServerManager', function () {
 
         it('should throw error when URL is invalid', async () => {
             let testee = fixture.createTestee();
+            let invalidURL = 'foobar';
             try {
-                await testee.getUpdateArchive({link: 'foobar'});
+                await testee.getUpdateArchive({link: invalidURL});
                 assert.fail('Expected error not thrown!');
             } catch(error) {
-                assert.equal(error.message, 'Invalid URL: foobar');
+                assert.equal(error.message + ':' + invalidURL, 'Invalid URL:'+ invalidURL);
```

Comment:
This assertion does not look correct to me.
The error message seems to not contain the URL anymore.
The old assertion looks correct to me, so the production code probably needs to be fixed and not the test.

Reply:
Hi @ronny1982!

The error is coming from node.js internals, not the repo of the application.
![image](https://github.com/manga-download/hakuneko/assets/51185703/6b7f20d6-8bc4-40ef-b149-61a10d8a3a9a)


I just finished double checking on this to be safe.
Changes were made on the Node.js internals to remove the invalidURL from the error message, please see below of the change that was made: https://github.com/nodejs/node/blob/d150316a8ecad1a9c20615ae62fcaf4f8d060dcc/lib/internal/errors.js#L1379


Thank you for the feedback!

Reply:
So this means the test output depends on the NodeJS version on which the test is performed?

In this case, the test should be adjusted to work with any NodeJS version and not just with the latest, or the test should be kept as is and the node version for this project needs to be locked to the same version that is shipped with the HakuNeko Desktop Client (Electron 8.3.4 => NodeJS 12.13.0)

Reply:
OK, it seems the build agents are installing NodeJS 16.x instead of the required 10.x which explains why the test is breaking now on the pipeline.

<img width="483" alt="image" src="https://github.com/manga-download/hakuneko/assets/27136773/b3871076-b3fb-4590-bd47-30e7208d94fd">

Reply:
Correct, they were 12.x previously. Those changes within the github actions workflow were made here, I THINK:
https://github.com/manga-download/hakuneko/pull/4949/commits/73826d5d9013128b23658727f374af718e74ba44

Reply:
I can adjust the test to work with either version, but first we need to decide, do we want to go back to 12.x on the github workflow as well, in that case I would double check to see if the issue even exists once we go back.

Reply:
Sorry, I didn't see the bottom of you message, my mistake.
Let me make a separate PR to change the build pipeline and double check the test then.